pkg/config/centralconfig.go

```diff
@@ -157,6 +169,7 @@ func NewCentralConfig(agentType AgentType) CentralConfig {
 		SubscriptionConfiguration: NewSubscriptionConfig(),
 		AppendDataPlaneToTitle:    true,
 		UpdateFromAPIServer:       false,
+		EventAggregationInterval:  1 * time.Minute,
```

Comment:
May be we should increase this default time to 5 or 10 mins?

Reply:
fixed